infrastructure/dpladm/bin/sync-site.sh

```diff
@@ -97,6 +97,70 @@ function getSiteSecondaryDomains {
     return
 }
 
+function projectHasGo {
+    local hasGo
+    hasGo=$(yq eval ".sites.${1}.hasGo" "${2}")
+    if [[ -z "${hasGo}" ]]; then
+        echo "false"
+        return  # Use 'exit' if not inside a function
+    fi
+    if [[ "${hasGo}" == "true" ]]; then
+        echo "true"
+        return
+    fi
+    if [[ "${hasGo}" == "false" ]]; then
+        echo "false"
+        return
+    fi
+    echo "false"
+    return
+}
```

Comment:
I would recommend only going with `true` as a true value - having several positive values only makes things more confusing when people see values being used in the yaml-file.

Reply:
Also - generally when making values-files - its only the boolean value that is used as a true value. The few times you use the string `"true"` is when the value is passed to another system directly, like eg. nginx annotations.